UI/adminhome.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <meta name="author" content="Michael Robert Wali">
+    <title>Admin</title>
+    <link href="css/style.css" type="text/css" rel="stylesheet">
+    <script type="text/javascript" src="js/main.js"></script>
+  </head>
+  <body>
+    <section class="adminlayout">
+      <nav>
+        <h1>Welcome Admin to <span class="caps">Fast Food Fast</span></h1>
+        <a href="index.html">Logout</a>
+      </nav>
+      <div id="adminmenu">
+          <h1>List of Orders</h1>
+          <div id="food">
+            <table>
+              <thead><tr><th>Food Order</th><th>Check</th></tr></thead>
+              <tbody>
+                <tr><td>Chips</td><td><input type="checkbox"></td></tr>
+                <tr><td>Kebabs</td><td><input type="checkbox"></td></tr>
+                <tr><td>Sausages</td><td><input type="checkbox"></td></tr>
+                <tr><td>Chapati</td><td><input type="checkbox"></td></tr>
+                <tr><td>Liver</td><td><input type="checkbox"></td></tr>
+                <tr><td>Pilau</td><td><input type="checkbox"></td></tr>
+                <tr><td>Goats Meat</td><td><input type="checkbox"></td></tr>
+                <tr><td>Matooke</td><td><input type="checkbox"></td></tr>
+                <tr><td>Plain Rice</td><td><input type="checkbox"></td></tr>
+                <tr><td>Pizza</td><td><input type="checkbox"></td></tr>
+                <tr><td>Samosa</td><td><input type="checkbox"></td></tr>
+                <tr><td>African Tea</td><td><input type="checkbox"></td></tr>
+                <tr><td>Beef</td><td><input type="checkbox"></td></tr>
+                <tr><td>Beans</td><td><input type="checkbox"></td></tr>
+                <tr><td>Fish</td><td><input type="checkbox"></td></tr>
+                <tr><td>Cow Peas</td><td><input type="checkbox"></td></tr>
+              </tbody>
+            </table>
+          </div>
+          <div id="order">
+            <button type="submit" class="button1" onClick="addToCart()" >Accept Order</button>
+            <button type="submit" class="button2" onclick="refreshPage()">Decline Order</button>
+          </div>
+          <div id="adminpic">
+            <img src="images/fastfood3.png">
+          </div>
+      </div>
+      <div id="admincart">
+        <h1>Mark Orders as Completed</h1>
+        <div id="selecteditems">
+          <table>
+            <tr><th></th></tr>
+          </table>
+          <button type="submit" class="button" onClick="clearDiv(); markOrder(); refreshPage();" >Complete</button>
+        </div>
+      </div>
+      <div id="admincart2">
+        <h1>List of Food Items</h1>
+        <div id="foodlist">
+          <table>
```

Comment:
Please use `<tr>` and `<td>` inside `<table>` and then you can add other HTML tags like`<p>`.